.env

```diff
@@ -3,3 +3,11 @@ MYSQL_USER=contenta
 MYSQL_PASSWORD=contenta
 MYSQL_ALLOW_EMPTY_PASSWORD=yes
 MYSQL_ROOT_PASSWORD=root
+
+HOSTNAME=contenta.local
```

Comment:
I think that for decoupled development, it's maybe good having localhost as the host so things like service workers and so on don't get blocked?

Reply:
That's a really good idea, as long as its on a port I guess. We really don't want to clobber other localhost apps that might be running on the host.